### PR TITLE
Some text changes

### PR DESCRIPTION
**Pull request by abdolg**
Added and changed some content. Only in german because we will translate the german content all in once

### CODE REVIEW

File: data/pages/data-privacy-de.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ language: de
 
 1. Im Folgenden informieren wir über die Erhebung personenbezogener Daten bei Nutzung unserer Website. Personenbezogene
    Daten sind alle Daten, die auf Sie persönlich beziehbar sind, z. B. Name, Adresse, E-Mail-Adressen, Nutzerverhalten.
-2. Verantwortlicher gem. Art. 4 Abs. 7 EU-Datenschutz-Grundverordnung (DS-GVO) ist Satellytes Digital Consulting GmbH,
-   Carola-Neher-Straße 10, 81379 München, gholam.abdol@satellytes.com (siehe [Impressum](/imprint/)).
+2. Verantwortlicher gem. Art. 4 Abs. 7 EU-Datenschutz-Grundverordnung (DSGVO) ist Satellytes Digital Consulting GmbH,
+   Carola-Neher-Straße 10, 81379 München, gholam.abdol@satellytes.com (siehe [Impressum](/de/imprint/)).
```
Comment (fischeversenker) [inline data/pages/data-privacy-de.md]: The `/de/` in the link should not be necessary. Looks like our markdown renderer does not take the current language into account. Fine for now but I will create an issue for this.

File: src/assets/locales/de/translations.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "main.description": "Wir sind pragmatische Profis, die zuverlässige Software für das Web entwickeln.",
-  "main.services.title": "Plattformen, Beratung & Produkte",
+  "main.description": "Wir machen die digitale Welt einfacher. Mit maßgeschneiderter Web-Software für Unternehmen. Von Frontend bis Backend. Vom Webauftritt bis zur digitalen Sicherheitsinfrastruktur.",
+  "main.services.title": "Wir verbinden. Mit Technologie.",
```
Comment (fischeversenker) [inline src/assets/locales/de/translations.json:3]: I love it 😍

File: src/assets/locales/de/translations.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   "main.services.kicker": "Unsere Leistungen",
-  "main.services.text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements im Konzerngeschäft. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen:",
+  "main.services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. \u2028Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
```
Comment (fischeversenker) [inline src/assets/locales/de/translations.json:5]: "\u2028" is not being displayed properly on my machine:
image

What was that supposed to be? A line break?

I also suggest to combine the last two sentences, like so:
```suggestion
  "main.services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. \u2028Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität und jeder Menge Herzblut.",
```

File: src/assets/locales/de/translations.json [hunk 4]
```diff
@@ -106,17 +106,17 @@
...
+  "about-us.team.text": "Unser erfahrenes Team aus Software Ingenieuren und Designern",
   "about-us.leadbox.title": "Haben Sie Interesse bekommen?",
   "about-us.leadbox.link": "Karriere",
   "services.hero": "Leistungen",
-  "services.title": "Unser Angebot an Sie",
+  "services.title": "Wir verbinden Development & Design",
   "services.kicker": "Plattformen, Produkt & Beratung",
-  "services.text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements von Mittelstand bis Konzerne. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen.",
+  "services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
```
Comment (fischeversenker) [inline src/assets/locales/de/translations.json]: This time without the strange unicode character but I'd like to suggest to combine the last two sentences here as well:
```suggestion
  "services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität und jeder Menge Herzblut.",
```

File: src/assets/locales/de/translations.json [hunk 5]
```diff
@@ -125,17 +125,18 @@
   "services.platform.list.designsystems": "Design Systeme",
   "services.platform.list.infrastructure": "Infrastruktur & Automation",
   "services.consulting.title": "Beratung",
-  "services.consulting.teaser": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
-  "services.consulting.text": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen diesen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
+  "services.consulting.teaser": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
+  "services.consulting.text": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen unser Wissen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
```
Comment (fischeversenker) [inline src/assets/locales/de/translations.json:132]: The `\n` line breaks don't seem to work.
image
Did they work in the past? Do we want them to work? If not we might as well remove them.

**Review by fischeversenker — APPROVED** [review]
Thanks for these changes. Everything looks great except one strange Unicode character. See my comments. I'm approving already so I don't block the PR, but please take care and remove that strange character.

File: data/pages/data-privacy-de.md (hunk 1 above)
Comment (milljoniaer) [reply data/pages/data-privacy-de.md]: I have made an already merged PR with de `/de/` workaround

**Pull request merged**